apps/gp2-server/test/data-providers/working-group.data-provider.test.ts

```diff
@@ -77,5 +78,160 @@ describe('Working Group Data Provider', () => {
         leadingMembers: '',
       });
     });
+
+    describe('Parsing', () => {
+      test('the working group is parsed', () => {
+        const workingGroup = getGraphQLWorkingGroup();
+        const workingGroupDataObject =
+          parseWorkingGroupToDataObject(workingGroup);
+        const expected = getWorkingGroupDataObject();
+        expect(workingGroupDataObject).toEqual(expected);
+      });
+
+      test('the members in working group are parsed', () => {
+        const workingGroup = getGraphQLWorkingGroup();
+        const workingGroupWithMembers = {
+          ...workingGroup,
+          flatData: {
+            ...workingGroup.flatData,
```

Comment:
you're trying to avoid mutations of fixtures in tests which imho seems unnecessary (we have created fixture factories specifically for this scenario) and now you are making it difficult for yourself cause you have to spread the object at different nesting levels, which also makes the fixture less readable. 

I would just mutate the members property of the `workingGroup` object

Reply:
Done